Build break on Ubuntu 14.04

When I tried to build servo on Ubuntu 14.04, I got build break. The error message showed below.
```
Compiling servo v0.0.1 (file:///media/vliu/workgroup/servo/servo/components/servo)
error: linking with `cc` failed: exit code: 1
note: "cc" "-Wl,--as-needed" "-m64" "-L" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/media/vliu/workgroup/servo/servo/target/release/servo.o" "-o" "/media/vliu/workgroup/servo/servo/target/release/servo" "-Wl,--whole-archive" "-l" "morestack" "-Wl,--no-whole-archive" "-Wl,--gc-sections" "-pie" "-Wl,-O1" "-nodefaultlibs" "/media/vliu/workgroup/servo/servo/target/release/libservo.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblayout-dc449b241f1fd5ff.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libprofile-5b30cea1e4986665.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libclock_ticks-4e910fa1778932df.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libenv_logger-9877a407b506c549.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libscript-6c5679f92a150148.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libcanvas-75503b8b00d94fd8.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libjs-1b3537edd7ef8c3c.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libwebsocket-a5de7834fe8fb871.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libwebdriver_server-773332ed82851371.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libwebdriver-26582494b13cc291.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libhtml5ever-1c899fee28a7ff79.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libmac-9253dcbe1dddc564.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libbyteorder-399c175f6a7726ac.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libglutin_app-5696dc16ceb2c43d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libcompositing-3178d2aa801b1c77.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libnet-36551dc93438c158.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libflate2-41314c0597876aa0.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libclipboard-e57e1abbe903eb91.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libglutin-da918fe8df5093ef.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblayout_traits-2fba9b46cb1d776f.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libregex-af3fdc791da043fa.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libregex_syntax-74cdf8e4e9018a0c.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libgfx-30d64c164ebd4bb5.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libscript_traits-f9b5108109642d62.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libnet_traits-4ad3538e4ae4b2de.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libfontconfig-98a34dc6c01f3190.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libstb_image-1f6702ea4c6749c2.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libprofile_traits-36bbb0599b45a246.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libharfbuzz-8689da4f9d543090.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libminiz_sys-9343607e1ddc701d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libuuid-e3cfaf8f9acfc716.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libcanvas_traits-154260479d29ed02.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liboffscreen_gl_context-c41d0deb91f407fa.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libgfx_traits-732276a4165cc6cb.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libdevtools-7ad6d724e23ca1f7.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libdevtools_traits-27533f07f1867eb2.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libmsg-ddde6fed44bd71b1.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libhyper-92eac666467560fa.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libhttparse-b6fd4407af2d76cd.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libcookie-e6818e9cbd9a23bd.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libunicase-1e872b36121cae26.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libstyle-ef96800eca29ccb8.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libutil-8b976d070614da49.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liburl-a74304d08ea0134d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblayers-847af0c24c0bb6f7.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libgleam-7cae7ac441cfe8c9.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libgl_common-2e67320e8d48721d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libazure-f792f70de9cc89e8.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libskia-d333fdba915552af.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libeuclid-237aff296570b536.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libnum-7ad397ad0b46ae38.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/librustc_serialize-c1e8163a38ed3d54.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libtime-10c3f659b0cce9f1.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libselectors-738a99aca96b392b.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libfnv-5893ac1ec1276f44.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libquicksort-6ca40ab3529e5e7a.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libstring_cache-4cb1b0fb2e281397.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libphf-a4b37fab7c341bfa.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libdebug_builders-337a04ff86d82ad7.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libstring_cache_shared-caccf3bc5141c0c3.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libfreetype-bc804dedae87cd10.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libtypeable-3d4cacf3175a54fa.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libpng-52685ef353c025b7.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libtest-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgetopts-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libterm-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libmime-89b55e354a702f26.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblog-8a6aba167994951e.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libnum_cpus-16707c6acca9fe91.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libsmallvec-a0ec234d91e55db4.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libserialize-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblog-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libcssparser-52f88f21cae7c8e6.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libmatches-68291f81832fc22d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding-a635ee1112a4a115.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding_index_japanese-02d1e3db3f81af39.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding_index_singlebyte-989206689effef1e.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding_index_tradchinese-1ea03524459844c3.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding_index_korean-e6ea16a131576deb.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libencoding_index_simpchinese-36d98fe1ee6656fc.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libphf_shared-ac2a5563029e2f49.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libbitflags-425420435a0b4460.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libx11-ed6c5bce2a89a9a8.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libtraitobject-e8fd97c9f52dc0fb.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/librand-de6cdb9e4fd93d55.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libopenssl-8dfa7204ef9d90f3.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libbitflags-dd68b8369bcd8ff0.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libopenssl_sys-9c1a0f13b3d0a12d.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblazy_static-0b01921545729662.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/libglx-3a94d6d02aa3bea2.rlib" "/media/vliu/workgroup/servo/servo/target/release/deps/liblibc-ef5cbad4ef5c7a1e.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcollections-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_unicode-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/librand-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-d8ace771.rlib" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-d8ace771.rlib" "-L" "/media/vliu/workgroup/servo/servo/target/release" "-L" "/media/vliu/workgroup/servo/servo/target/release/deps" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/azure-f792f70de9cc89e8/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/expat-sys-e22582e7d0e6b2e6/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/fontconfig-sys-1f38cf9d1e1a4d19/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/freetype-sys-6942b42598677f74/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/harfbuzz-8689da4f9d543090/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/js-1b3537edd7ef8c3c/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/miniz-sys-9343607e1ddc701d/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/mozjs_sys-e411d274df144795/out/dist/lib" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/openssl-sys-9c1a0f13b3d0a12d/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/png-52685ef353c025b7/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/png-sys-c0fc68e5ccc34308/out/.libs" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/skia-d333fdba915552af/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/stb_image-1f6702ea4c6749c2/out" "-L" "/media/vliu/workgroup/servo/servo/target/release/build/time-10c3f659b0cce9f1/out" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/usr/lib/x86_64-linux-gnu" "-L" "/media/vliu/workgroup/servo/servo/.servo/rust/6e7fcc44aef7b457f3be3a1971d9f026957678d5/rustc-1.2.0-dev-x86_64-unknown-linux-gnu/rustc/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-L" "/media/vliu/workgroup/servo/servo/components/servo/.rust/lib/x86_64-unknown-linux-gnu" "-L" "/media/vliu/workgroup/servo/servo/components/servo/lib/x86_64-unknown-linux-gnu" "-Wl,-Bstatic" "-Wl,-Bdynamic" "-l" "rt" "-l" "pthread" "-l" "stdc++" "-l" "z" "-l" "jsglue" "-l" "c" "-l" "fontconfig" "-l" "c" "-l" "glib-2.0" "-l" "stdc++" "-l" "GL" "-l" "stdc++" "-l" "freetype" "-l" "bz2" "-l" "fontconfig" "-l" "expat" "-l" "X11" "-l" "azure" "-l" "skia" "-l" "rt" "-l" "freetype" "-l" "z" "-l" "Xcursor" "-l" "Xxf86vm" "-l" "X11" "-l" "ssl" "-l" "crypto" "-l" "c" "-l" "m" "-l" "dl" "-l" "pthread" "-l" "rt" "-l" "gcc_s" "-l" "pthread" "-l" "c" "-l" "m" "-l" "compiler-rt"
note: /media/vliu/workgroup/servo/servo/target/release/deps/libclipboard-e57e1abbe903eb91.rlib(clipboard-e57e1abbe903eb91.o): In function `x11_clipboard::ClipboardContext::new::h74a404002a4ba9aaEaa':
clipboard.0.rs:(.text._ZN13x11_clipboard16ClipboardContext3new20h74a404002a4ba9aaEaaE+0xa9): undefined reference to `_XA_CLIPBOARD'
clipboard.0.rs:(.text._ZN13x11_clipboard16ClipboardContext3new20h74a404002a4ba9aaEaaE+0xb4): undefined reference to `XmuInternAtom'
clipboard.0.rs:(.text._ZN13x11_clipboard16ClipboardContext3new20h74a404002a4ba9aaEaaE+0xc7): undefined reference to `_XA_UTF8_STRING'
clipboard.0.rs:(.text._ZN13x11_clipboard16ClipboardContext3new20h74a404002a4ba9aaEaaE+0xd2): undefined reference to `XmuInternAtom'
collect2: error: ld returned 1 exit status

error: aborting due to previous error
Could not compile `servo`.
```

`note: /bin/ld.gold: error: cannot find -lEGL`. You need to install the appropriate package; it should be `mesa-libgl-devel` if you're on Fedora I think.

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ On Fedora:
 
 ``` sh
 sudo dnf install curl freeglut-devel libtool gcc-c++ libXi-devel \
-    freetype-devel mesa-libGL-devel glib2-devel libX11-devel libXrandr-devel gperf \
+    freetype-devel mesa-libGL-devel mesa-libEGL-devel glib2-devel libX11-devel libXrandr-devel gperf \
     fontconfig-devel cabextract ttmkfdir python python-virtualenv python-pip expat-devel \
     rpm-build openssl-devel cmake bzip2-devel libXcursor-devel libXmu-devel mesa-libOSMesa-devel
 ```

